### DISCUSSION TITLE
Zoom max in titiler?

### DISCUSSION THREAD

**Discussion by emmanuelmathot** [developmentseed/titiler#248] [Q&A]
Hello, we have an issue with some multi band images (RGBA) when zooming into the raster. After a certain zooming level (probably after the native resolution), the image disappears. Titiler does not return any error, just a blank transparent image.
The raster image is properly cogified with full res data and with overviews. I wonder about a kind of ZOOM_MAX parameter somewhere in the image. If you have any hint about the issue, please let me know.
Thank you!

image is transformed in COG this way

```python
translate_options = gdal.TranslateOptions(
    gdal.ParseCommandLine(
        "-co TILED=YES " "-co COPY_SRC_OVERVIEWS=YES " "-co COMPRESS=DEFLATE "
    )
)

ds = gdal.OpenEx(input_tif, gdal.OF_RASTER | gdal.OF_READONLY)

gdal.SetConfigOption("COMPRESS_OVERVIEW", "DEFLATE")
ds.BuildOverviews("NEAREST", [2, 4, 8, 16, 32, 64])
ds = None
del ds
ds = gdal.Open(input_tif)

if output_type is not None:
  gdal.Translate(output_tif, ds, options=translate_options)
```

**Comment by vincentsarago** [comment]
👋 @emmanuelmathot 

> After a certain zooming level (probably after the native resolution), the image disappears. 

This sounds like a frontend issue, not really linked with TiTiler. A quick check with rio-tiler confirmed that we should still be able to create tile way after the max-zoom.

```python
from random import randint, sample
from rio_tiler.io import COGReader
from supermercado.burntiles import tile_extrema


with COGReader("https://rio-tiler-dev.s3.amazonaws.com/data/m_3209542_se_15_060_20181115_20190222.tif") as cog:
    ovrzoom = 4
    extrema = tile_extrema(cog.bounds, cog.maxzoom + ovrzoom)
    tile_x = sample(range(extrema["x"]["min"], extrema["x"]["max"]), 1)[0]
    tile_y = sample(range(extrema["y"]["min"], extrema["y"]["max"]), 1)[0]
    tile_z = cog.maxzoom + ovrzoom
    img = cog.tile(tile_x, tile_y, tile_z)
    assert img.data.any()
    assert img.mask.any()
``` 


> Titiler does not return any error, just a blank transparent image.

are you certain that Titiler returns something?

**Reply by kylebarron** [reply]
I also haven't had issues in the past with fetching upsampled data at higher resolutions from titiler.

**Comment by LanesGood** [comment]
@emmanuelmathot did you have continued issues with this? We've surfaced a similar error on another project and have tracked it down to the frontend - would be interested in comparing notes if you recall what happened here.

**Reply by emmanuelmathot** [reply]
That's pretty old and I do not remember the exact application. 
Maybe @francescocerasuolo can help with this. Something with leaflet in the geobrowser? I just recall having debugged it directly in titiler calls.

> We've surfaced a similar error on another project and have tracked it down to the frontend

@LanesGood Does it mean the cause is on the frontend side?

**Reply by vincentsarago** [reply]
Titiler/rio-tiler should always return overzoomed tiles. I'm pretty sure issues always come down to the min/max zoom or min/max NativeZoom settings in frontends
